docs/develop/01_sdk/04_integrate/02_browser.md

```diff
@@ -3,7 +3,45 @@ id: howto-integrate-browser
 title: Browser
 ---
 
-Integrating the KILT SDK into a browser application involves "tricking" the browser into believing it is running on a Node server.
-This is needed for the advanced cryptographic primitives to work properly.
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-To simplify the setup, we recommend using the [KILT Distillery](./03_distillery.md) or something like [ViteJS](./04_vitejs.md) to bootstrap a project using the KILT SDK.
+Our javascript SDK (`@kiltprotocol/sdk-js`) is ready to be used in a browser context. For rapid prototyping of simple web apps, we provide a code bundle of the entire SDK which you can add embed in a site by adding the following script tag:
+
+```html
+<script src="https://unpkg.com/@kiltprotocol/sdk-js@0.33.1/dist/sdk-js.min.umd.js"></script>
+```
+
+The SDK's functions then become available via a new `kilt` property on the global `window` object.
+
+To get started with your first __React application__ using Kilt, we recommend using either the [KILT Distillery](./03_distillery.md) CLI tool or bootstrapping tools like [Create React App](https://create-react-app.dev) or [ViteJS](https://vitejs.dev) that take away some of the complexity in building and testing a React application.
```

Comment:
I would really think twice about recommending CRA as a modern tool. They don't even mention it in the official [React docs](https://react.dev/) anymore

Reply:
oh okay, didn't know that. So people just use viteJS now? Is there another tool that's used instead?

Reply:
Should we recommend NextJS instead?

Reply:
Yeah, Vite is the most popular quick-start option these days. Maybe we could also link to React's recommended frameworks:

https://react.dev/learn/start-a-new-react-project

Reply:
how's [that](https://github.com/KILTprotocol/docs/pull/257/commits/bbc9ff7de1789258e284f5e7a18163158f82e47e)?

Reply:
Looks good!